File: LeetCode_Marathons/LCM5/Easy/Missing_Number/a.go
func missingNumber(nums []int) int {
    var needSumOfNums, sumOfNums int
    n := len(nums)

    needSumOfNums = n * (n + 1) / 2;
    for _, num := range(nums) {
        sumOfNums += num
    }

    return needSumOfNums - sumOfNums;
}

File: LeetCode_Marathons/LCM5/Easy/Missing_Number/a.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int needSumOfNums = 0, sumOfNums = 0;
        int n = nums.size();

        needSumOfNums = n * (n + 1) / 2;
        for (int i = 0; i < n; i++) sumOfNums += nums[i];

        return needSumOfNums - sumOfNums;
    }
};